sdk/python/tests/dsl/pipeline_param_tests.py

```diff
@@ -104,3 +104,17 @@ def test_extract_pipelineparam_with_types(self):
         ]
         params = _extract_pipelineparams(payload)
         self.assertListEqual([p1, p2, p3], params)
+
+class TestSanitizeK8sName(unittest.TestCase):
+
+    def test_argo_variables(self):
+        cases = ["{{workflow.name}}", "abc-{{workflow.name}}-def..", "{{workflow.name}}-?{{workflow.uid}}"]
+        got = list(map(sanitize_k8s_name, cases))
+        expected = ["{{workflow.name}}", "abc-{{workflow.name}}-def", "{{workflow.name}}-{{workflow.uid}}"]
+        self.assertListEqual(got, expected)
+
+    def test_sanitize_names(self):
+        cases = ["abc*def*", "abcAbc-_-"]
+        got = list(map(sanitize_k8s_name,cases))
+        expected = ["abc-def", "abc-bc"]
```

Comment:
```suggestion
        cases = ["abc*def*", "abcAbc-_-"]
        got = list(map(sanitize_k8s_name,cases))
        expected = ["abc-def", "abcabc"]
```

Furthermore ,regarding Sanitization I feel abcAbc -> abcabc is better over abc-bc. Should I make changes for this ?

Reply:
> I feel abcAbc -> abcabc is better over abc-bc

+1